app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments
+  has_many :likes
```

Comment:
- [ ] User has many comments and likes through the `author_id` column. Let's specify that to avoid Rails looking for a `user_id`.

  ```
  has_many :comments, foreign_key: 'author_id'
  has_many :likes, foreign_key: 'author_id'
  ```